examples/case-study-1/load-db.p6

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env perl6
+
+use Red;
+
+
+use lib <./lib>;
+use OurFuncs;
+
+=begin comment
+# do not know how to 'use' these yet:
+#model Person {...}
+use Present;
+use Attend;
+use Email;
+use Person;
+=end comment
+
+#==========================================================
+# TODO Successfully split the following models into a lib
+#      directory as individual packages and 'use' them
+#      in this script and the 'query-db.p6' script.
+model Person {...}
+
+model Email is rw {
+    # relationship
+    has UInt $!person-id is referencing{:column<key>, :model<Person>};
+    has Person $.person  is relationship{:column<key>, :model<Person>};
+
+    # data:
+    has Str  $.email     is column;
+    has Str  $.notes     is column{:nullable};
+    has UInt $.status    is column{:nullable};
+}
+
+model Present is rw {
+    # relationship
+    has UInt $!person-id is referencing{:column<key>, :model<Person>};
+    has Person $.person  is relationship{:column<key>, :model<Person>};
+
+    # data:
+    has UInt $.year      is column;
+    has Str  $.notes     is column{:nullable};
+}
+
+model Attend is rw {
+    # relationship
+    has UInt $!person-id is referencing{:column<key>, :model<Person>};
+    has Person $.person  is relationship{:column<key>, :model<Person>};
+
+    # data:
+    has UInt $.year      is column;
+    has Str  $.notes     is column{:nullable};
+}
+
+model Person is rw {
+    has Str $.key          is column{:unique};
```

Comment:
To use `.^load` currently it should be a pk... try with `is id`

Reply:
That doesn't work, maybe because key is a string. Using grep for now.